db/seeds.rb

```diff
@@ -18,6 +18,13 @@ def format_errors(model = nil)
     ForemanTasks.sync_task(::Actions::Katello::Organization::Create, org) unless org.library
     User.current = nil
   end
+  if Setting['db_pending_seed']
+    admin = User.where(:login => ENV['SEED_ADMIN_USER'].present? ? ENV['SEED_ADMIN_USER'] : 'admin').first
+    if admin && admin.default_organization.nil?
+      admin.default_organization = Organization.find_by_name!(ENV['SEED_ORGANIZATION'])
+      admin.save!
+    end
+  end
```

Comment:
This could, if an admin has purposefully unset or kept unset their default organization, result in that admin having one set for them? Might be inconsequential, but worth noting in the release notes for upgrades.